modules/tinymce/src/themes/silver/main/ts/Render.ts

```diff
@@ -425,8 +425,12 @@ const setup = (editor: Editor): RenderInfo => {
       OuterContainer.focusToolbar(outerContainer);
     });
 
-    editor.addCommand('ToggleToolbarDrawer', () => {
-      OuterContainer.toggleToolbarDrawer(outerContainer);
+    editor.addCommand('ToggleToolbarDrawer', (_ui, options?: { skipFocus: boolean }) => {
+      if (options?.skipFocus) {
```

Comment:
You could also use a ternary here. I like ternaries because they force a value regardless of the branching. But I might be in the minority there. Up to you.

Reply:
I like ternaries too :) However we are not going to return any value here, so I'd leave the if statement.